packages/@okta/vuepress-site/docs/guides/migrate-to-okta-password-hooks/main/index.md

```diff
@@ -1,5 +1,26 @@
 ---
 title: Import Users with Inline Password Hooks
+meta:
+  - name: description
+    content: Migrate users into Okta as they authenticate using Password Import Inline Hooks
+layout: Guides
+---
+
+This guide explains how to migrate users into Okta as they authenticate using Password Import Inline Hooks
+
+---
+
+**Learning outcomes**
+
+Migrate users into Okta as they authenticate using Password Import Inline Hooks.
+
+**What you need**
+
+* An Okta developer org. (Don’t have one? [Create an org for free.](https://developer.okta.com/signup))
+* Postman client to run API requests. See [Use Postman with the Okta REST APIs](https://developer.okta.com/code/rest/) for information on setting up Postman.
+* Example or test source data to test user and group creation requests. (Do not use real user data when testing!)
+* [A plan for migrating existing users to Okta](/docs/guides/migrate-to-okta-prerequisites/).
+
```

Comment:
Does this guide merit a "Sample code" section here? I think that devs might appreciate knowing where to find the request code on the page.